### PR TITLE
[Notifier] Mention `postAt()` in the Slack bridge README

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix https://github.com/symfony/symfony-docs/issues/18958
| License       | MIT

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. I think we should mention this. But maybe do it in the Symfony-docs repo instead. 

The readmes here are intentionally very very short and contains basic info only.

**Comment by alexandre-daubois** [comment]
If I remember correctly, bridges' documentation are always contained in their own README, the doc doesn't mention available options of the different bridges. Maybe @OskarStark can confirm this as the code owner?

**Comment by Nyholm** [comment]
Yepp. You are correct. A quick look over the Notifier's Bridges show that Im wrong. Sorry about that.

File: src/Symfony/Component/Notifier/Bridge/Slack/README.md [hunk 1]
```diff
@@ -226,6 +226,25 @@ $options = new UpdateMessageSlackOptions($channelId, $messageId);
...
+```php
+use Symfony\Component\Notifier\Bridge\Slack\SlackOptions;
+use Symfony\Component\Notifier\Message\ChatMessage;
+
+$chatMessage = new ChatMessage('Symfony Feature');
+
+$options = (new SlackOptions())->postAt(new \DateTime('+1 day'));
+
+// Add the custom options to the chat message and schedule the message
+$chatMessage->options($options);
```
Comment (Nyholm) [inline src/Symfony/Component/Notifier/Bridge/Slack/README.md]: Would it be more clear if we re-orderd the lines?

```suggestion
$options = (new SlackOptions())->postAt(new \DateTime('+1 day'));

$chatMessage = new ChatMessage('Symfony Feature');
$chatMessage->options($options);
```

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by alexandre-daubois** [comment]
No worries :smile: I agree about the lines order, that's updated!

**Review by Nyholm — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alexandre-daubois.

**Pull request merged**